test/lit/passes/cfp.wast

```diff
@@ -771,17 +824,15 @@
 )
 
 ;; Subtyping: Create both a subtype and a supertype, with different constants
-;;            for the shared field, but get from the subtype. As the field is
-;;            shared between the types, we cannot optimize here: we model the
-;;            struct.new as a set, and if there is a set of the supertype, then
-;;            it might be to a reference of the subtype.
-;;            TODO: model new and set separately
+;;            for the shared field, but get from the subtype. The field is
+;;            shared between the types, but we only create the substruct with
+;;            one value, so we can optimize.
 (module
+  ;; CHECK:      (type $none_=>_none (func))
```

Comment:
It would be good to reorder the types in the input to match the output both here and below.

Reply:
Ah, thanks, looks like I missed a few...